tests/data/params.yaml

```diff
@@ -0,0 +1 @@
+ecs: 3
```

Comment:
`.yml` instead of `.yaml` as file ending?

Reply:
Don't care, shouldn't matter ... but:
> Is there an official extension for YAML files?
>
> Please use ".yaml" when possible.

http://yaml.org/faq.html

Reply:
Oh, ok. Didn't know that. Then, let's stick to `.yaml`.